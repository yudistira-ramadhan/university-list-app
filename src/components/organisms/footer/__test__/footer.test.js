import React from 'react'
import ReactDOM from 'react-dom'
import Footer from '../footer'
import { render } from '@testing-library/react'
import renderer from 'react-test-renderer'

it('renders without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(<Footer/>, div)
})

it('renders footer correctly', () => {
  const { getByTestId } = render(<Footer />)
  expect(getByTestId('footer')).toBeTruthy()
})

it('matches snapshot', () => {
  const tree = renderer.create(<Footer />).toJSON()
  expect(tree).toMatchSnapshot()
})