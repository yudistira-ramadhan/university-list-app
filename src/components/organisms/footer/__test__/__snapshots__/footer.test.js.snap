// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="container-fluid"
>
  <div
    className="footer__copyright"
    data-testid="footer"
  >
    All Rights Reserved Â© 
    2020
     by Yudistira Ramadhan
  </div>
</div>
`;
